animation_nodes/data_structures/attributes/attribute.pxd

```diff
@@ -0,0 +1,14 @@
+cdef enum AttributeType:
+    UVMAP, MATERIAL_INDEX, VERTEX_COLOR, CUSTOM
+cdef enum AttributeDomain:
+    POINT, EDGE, POLYGON, CORNER
+cdef enum AttributeDataType:
+    INT, FLOAT, FLOAT2, FLOAT_VECTOR, FLOAT_COLOR, BYTE_COLOR, BOOLEAN
+
+cdef class Attribute:
+    cdef:
+        public str name
+        public AttributeType type
+        public AttributeDomain domain
+        public AttributeDataType dataType
+        public object data
```

Comment:
Can this be `CList` of is `object` necessary?

Reply:
Removed.